### ISSUE TITLE
`--comment-descriptions` does not work anymore with v3

### ISSUE THREAD

**Issue by yvele** [cjoudrey/graphql-schema-linter#320]
I switched from graphql-schema-linter v2.0.2 to v3.0.1

And now I have a lot of `types-have-descriptions` and `fields-have-descriptions` reports.

This should be OK using `--comment-descriptions`:

```graphql
# Foo description.
type Foo {

  # Unique identifier.
  id: ID!

}
```

Everything was OK using v2.0.2

Note that I'm building a schema for AWS AppSync and I need the old way of defining descriptions:
- https://github.com/aws/aws-appsync-community/issues/38

**Comment by deechauhan** [comment]
+1

I'm using AWS AppSync - Although it's annoying why AWS still doesn't support proper descriptions, would be amazing if `--comment-descriptions` worked.
